### ISSUE TITLE
Expose the transferred number of bytes

### ISSUE THREAD

**Issue by tillkruss** [phpredis/phpredis#2106]
From Slack discussions.

It would be handy to have access to a `getTransferredBytes()` (or similar) method that would expose the total number of bytes transferred in the last call.

```php
$redis->ping();
$redis->getTransferredBytes(); // 6 bytes

$redis->hgetall('foo');
$redis->getTransferredBytes(); // 1337 bytes
```

**Comment by yatsukhnenko** [comment]
@tillkruss How do you get 6 bytes for `ping`? Raw command looks like `*1\r\n$4\r\nPING\r\n` and response is `+PONG\r\n`

**Comment by tillkruss** [comment]
@yatsukhnenko I made that number up, just to signal that even `ping()` would cause transferred bytes.

**Comment by yatsukhnenko** [comment]
So this should be the number of sent bytes, right?

**Comment by tillkruss** [comment]
Yeah, just of the last executed command. No aggregate of any sort.

**Comment by yatsukhnenko** [comment]
@tillkruss could you test `issue-2106` branch?

**Comment by xiaoxiongjie** [comment]
您的邮件已经收到

**Comment by yatsukhnenko** [comment]
@tillkruss @michael-grunder I guess we can close this issue?

**Comment by tillkruss** [comment]
Yeah, I think Michael will rename the methods to match Relay, but yes.
